airflow/contrib/hooks/sqoop_hook.py

```diff
@@ -145,7 +143,8 @@ def _prepare_command(self, export=False):
 
         return connection_cmd
 
-    def _get_export_format_argument(self, file_type='text'):
+    @staticmethod
+    def _get_export_format_argument(file_type='text'):
```

Comment:
Function doesn't use `self` and should be static

Reply:
👍